src/main/java/bubbles/Storage.java

```diff
@@ -0,0 +1,109 @@
+package bubbles;
+
+import bubbles.tasks.Task;
+import bubbles.tasks.TaskList;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * A class that deals with loading bubbles.tasks from the file
+ * saved on the hard disk, followed by saving the tasks in the Bubbles bot
+ * back into the file after the program ends.
+ */
+public class Storage {
+    private File bubbles;
+    private Parser parser;
+    private TaskList taskList;
+
+    /**
+     * A public constructor to initialise a Storage Object for the
+     * Bubbles bot. Each Storage Object has the File that it reads from and writes to,
+     * the parser that decodes the user's inputs, and a task list for the Bubbles bot.
+     */
+    public Storage() {
+        this.parser = new Parser();
+    }
+
+    /**
+     * Loads the reference file of the Storage Object based on the filePath
+     * argument provided.
+     *
+     * @param filePath The path to the designated file with respect to the root working
+     *                 directory
+     */
+    public void loadFile(String filePath) {
+        bubbles = new File(filePath);
+
+        try {
+            taskList = readFile();
+        } catch (FileNotFoundException e) {
+            createFile(filePath);
+            taskList = new TaskList();
+        }
+    }
+
+    private void createFile(String filePath) {
+        String[] arr = filePath.split("/");
+
+        File file = new File(filePath);
+        File dir = new File(arr[0]);
+        dir.mkdir();
+        try {
+            file.createNewFile();
+        } catch (IOException e) {
+            System.out.println("There is an error while creating the database!");
+        }
+    }
+
+    private TaskList readFile() throws FileNotFoundException {
+        Scanner sc = new Scanner(bubbles);
+        TaskList taskList = new TaskList();
+
+        while (sc.hasNext()) {
+            String task = sc.nextLine();
+            Object[] inputs = parser.formatTask(task);
+
+            taskList.addTask((String) inputs[0], (String) inputs[1], (Boolean) inputs[2]);
```

Comment:
Perhaps a comment here specifying what are inputs[0], inputs[1], inputs[2] would be good? It'd be easier to refer to since Parser is in a separate file.